integration/cloudbuild/testbase/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# This Dockerfile builds a base image for the Trillan CloudBuild integration testing.
+FROM ubuntu as trillian_testbase
+
+WORKDIR /testbase
+
+ARG GOFLAGS=""
+ENV GOFLAGS=$GOFLAGS
+ENV GO111MODULE=on
+ENV GOPATH /go
+
+RUN apt-get update && apt-get -y install build-essential curl docker-compose lsof mysql-client socat unzip wget xxd softhsm
+
+RUN curl -sfL https://golang.org/dl/go1.15.7.linux-amd64.tar.gz | tar -xzf - -C /usr/local
+ENV PATH /usr/local/go/bin:$PATH
+
+RUN curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin v1.18.0
+RUN mkdir protoc && \
+    (cd protoc && \
+    PROTOC_VERSION="3.11.4" && \
+    PROTOC_ZIP="protoc-${PROTOC_VERSION}-linux-x86_64.zip" && \
+    wget "https://github.com/google/protobuf/releases/download/v${PROTOC_VERSION}/${PROTOC_ZIP}" && \
```

Comment:
This corresponds to this repo: https://github.com/protocolbuffers/protobuf

The "go import" in the other file imports from a different repo: https://github.com/golang/protobuf.

I guess, we need to make sure they are compatible. I.e. we need to "go import" from a version not far off in time from protoc v3.11.4.

Reply:
Hmm, I think the first repo gives us the `protoc` command, but the second is the golang specific bits (including `protoc-gen-go`).  You may be onto something with the golang/protobuf version but the `pb/protobuf` version is as it was in the [travis config](https://github.com/google/trillian/blob/master/.travis.yml#L40)